code/modules/vehicles/vehicle_actions.dm

```diff
@@ -192,7 +192,7 @@
 /datum/action/vehicle/action_activate()
 	if(HAS_TRAIT(owner, TRAIT_IMMOBILE))
 		return FALSE
-	if(!owner.stat)
+	if(owner.stat)
```

Comment:
??

Reply:
Bug I found while doing action buttons. It was doing the opposite effect, letting only players that were not conscious use actions.